src/set_theory/game/pgame.lean

```diff
@@ -228,6 +238,34 @@ meta def pgame_wf_tac :=
    subsequent.mk_left, subsequent.mk_right, subsequent.trans]
   { max_depth := 6 }]
 
+/-- The game `P{x | y}` with indexing types `punit`. -/
+@[reducible] def of (x y : pgame) : pgame := ⟨punit, punit, λ _, x, λ _, y⟩
+
+/-- The game `P{x |}` with indexing types `punit` and `pempty`. -/
+@[reducible] def of_left (x : pgame) : pgame := ⟨punit, pempty, λ _, x, pempty.elim⟩
+
+/-- The game `P{| x}` with indexing types `pempty` and `punit`. -/
+@[reducible] def of_right (x : pgame) : pgame := ⟨pempty, punit, pempty.elim, λ _, x⟩
+
+notation `P{` x ` | ` y `}` := of x y
```

Comment:
I think you have to use a fully-qualified name here, else it will fail if used without the namespace open:
```suggestion
notation `P{` x ` | ` y `}` := pgame.of x y
```
That might only be true of `localized notation` though.

Reply:
Better safe than sorry.